hlx_statics/blocks/inline-nested-alert/inline-nested-alert.js

```diff
@@ -0,0 +1,129 @@
+import { createTag, decorateAnchorLink } from "../../scripts/lib-adobeio.js";
+
+function addInfoSvg() {
+    return `
+        <svg
+            class="spectrum-Icon info-NestedAlertIcon nested-svg"
+            <path
+                d="M11 2a9 9 0 109 9 9 9 0 00-9-9zm-.15 2.65a1.359 1.359 0 011.431 1.283q.004.064.001.129a1.332 1.332 0 01-1.432 1.432 1.353 1.353 0 01-1.432-1.433 1.359 1.359 0 011.304-1.412q.064-.002.128.001zM13.5 16a.5.5 0 01-.5.5H9a.5.5 0 01-.5-.5v-1a.5.5 0 01.5-.5h1v-4H9a.5.5 0 01-.5-.5V9a.5.5 0 01.5-.5h2.5a.5.5 0 01.5.5v5.5h1a.5.5 0 01.5.5z"></path>
+            <path
+                d="M9 1a8 8 0 108 8 8 8 0 00-8-8zm-.15 2.15a1.359 1.359 0 011.431 1.283q.004.064.001.129A1.332 1.332 0 018.85 5.994a1.353 1.353 0 01-1.432-1.433 1.359 1.359 0 011.304-1.412q.064-.002.128.001zM11 13.5a.5.5 0 01-.5.5h-3a.5.5 0 01-.5-.5v-1a.5.5 0 01.5-.5H8V9h-.5a.5.5 0 01-.5-.5v-1a.5.5 0 01.5-.5h2a.5.5 0 01.5.5V12h.5a.5.5 0 01.5.5z"></path>
+        </svg>`
+}
+
+function addSuccessSvg() {
+    return `
+        <svg
+            class="spectrum-Icon success-NestedAlertIcon nested-svg"
+            <path
```

Comment:
Similar to: https://github.com/adobe/adobe-io-website/blob/fresh-spectrum-less/hlx_statics/blocks/inlinealert/inlinealert.js#L26-L31